### PR TITLE
Spanish translation: some corrections

### PR DESCRIPTION
**Pull request by hiperiondev**
compared against Italian translation

### CODE REVIEW

File: Translations/es.po [hunk 1]
```diff
@@ -152,7 +150,7 @@ msgstr "Director Provisional de Control de Calidad (Lead Tester)"
 #: Source/DiabloUI/credits_lines.cpp:115
 #, fuzzy
 msgid "QA Assault Team (Testers)"
-msgstr "Equipo de Control de Calidad (Testers)"
+msgstr "Escuadrón de Asalto de Control de Calidad (Testers)"
```
Comment (NikoVP) [inline Translations/es.po:153]:
```suggestion
msgstr "Equipo de Control de Calidad (Testers)"
```

File: Translations/es.po [hunk 2]
```diff
@@ -180,11 +178,10 @@ msgstr "Atención al cliente"
 msgid "Sales"
 msgstr "Ventas"
 
-# no se que es esto
+# https://english.stackexchange.com/questions/550254/does-anyone-have-know-the-etymology-of-dunsel
 #: Source/DiabloUI/credits_lines.cpp:142
-#, fuzzy
 msgid "Dunsel"
-msgstr "Dunsel"
+msgstr "Inútil"
```
Comment (NikoVP) [inline Translations/es.po:184]:
```suggestion
msgstr "Dunsel"
```

File: Translations/es.po [hunk 3]
```diff
@@ -4984,11 +4896,11 @@ msgstr "El Profanador"
 msgid "Bonehead Keenaxe"
 msgstr "Hacha Afilada de Cabeza de Hueso"
```
[the remark this answers is not in the document]
Comment (Cowcat5150) [reply Translations/es.po:4897]: Hacha afilada. Si es la descripción de un objeto es mejor el antiguo texto. "Hacha afilada de ..."

**Comment by AJenbo** [comment]
I'm not sure what the conversations here are resulting in, but unless you guys suggest other wise I will merge this as is later today, since there are many changes and only a few comments.

**Comment by hiperiondev** [comment]
Ok for me. 
I really am with a lot of work now and limited delivery times .

**Comment by NikoVP** [comment]
I guess it can be merged as it fixes a lot of stuff, still there thats that need further revision but can be done in different PR.

**Comment by Mad-Soft** [comment]
¿Con la versión 1.2.1 no va la traducción verdad?
¿En qué versión se espera que se de soporte?
¿Se puede obtener una versión beta para Linux/Windows que soporte la traducción?

**Comment by AJenbo** [comment]
1.3.0 will be out in October with transition support, you can download a test version from the Frontpage.

**Comment by AJenbo** [comment]
@hiperiondev all planned features affecting the Spanish translation should now be done and all translated text should show correctly in game. So now would be an excellent time to work on the renaming parts of the translation :)

**Comment by hiperiondev** [comment]
I have a lot of work. I think next week I would have more time and could go back on the translation

**Comment by AJenbo** [comment]
That sounds grate, we hope to wrap up the next release in 2 - 3 weeks

**Pull request merged**